components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -1553,36 +1553,58 @@
     public static final String GET_API_RATING_SQL =
             "SELECT RATING FROM AM_API_RATINGS WHERE API_ID= ? AND SUBSCRIBER_ID=? ";
 
+    public static final String GET_API_PRODUCT_RATING_SQL =
+            "SELECT RATING FROM AM_API_RATINGS WHERE API_PRODUCT_ID= ? AND SUBSCRIBER_ID=? ";
+
     public static final String ADD_API_RATING_SQL =
             "INSERT INTO AM_API_RATINGS (RATING_ID, RATING, API_ID, SUBSCRIBER_ID)  VALUES (?,?,?,?)";
 
+    public static final String ADD_API_PRODUCT_RATING_SQL =
+            "INSERT INTO AM_API_RATINGS (RATING_ID, RATING, API_PRODUCT_ID, SUBSCRIBER_ID)  VALUES (?,?,?,?)";
+
     public static final String UPDATE_API_RATING_SQL =
             "UPDATE AM_API_RATINGS SET RATING=? WHERE API_ID= ? AND SUBSCRIBER_ID=?";
 
-    public static final String GET_RATING_ID_SQL =
+    public static final String UPDATE_API_PRODUCT_RATING_SQL =
+            "UPDATE AM_API_RATINGS SET RATING=? WHERE API_PRODUCT_ID= ? AND SUBSCRIBER_ID=?";
+
+    public static final String GET_API_RATING_ID_SQL =
             "SELECT RATING_ID FROM AM_API_RATINGS WHERE API_ID= ? AND SUBSCRIBER_ID=? ";
 
+    public static final String GET_API_PRODUCT_RATING_ID_SQL =
+            "SELECT RATING_ID FROM AM_API_RATINGS WHERE API_PRODUCT_ID= ? AND SUBSCRIBER_ID=? ";
+
     public static final String REMOVE_RATING_SQL =
             "DELETE FROM AM_API_RATINGS WHERE RATING_ID =? ";
 
-    public static final String GET_RATING_SQL =
-            "SELECT RATING FROM AM_API_RATINGS WHERE SUBSCRIBER_ID  = ? AND API_ID= ? ";
-
-    public static final String GET_RATING_INFO_SQL =
+    public static final String GET_API_RATING_INFO_SQL =
             "SELECT RATING_ID, API_ID, RATING, SUBSCRIBER_ID FROM AM_API_RATINGS WHERE SUBSCRIBER_ID  = ? "
                     + "AND API_ID= ? ";
 
+    public static final String GET_API_PRODUCT_RATING_INFO_SQL =
+            "SELECT RATING_ID, API_PRODUCT_ID, RATING, SUBSCRIBER_ID FROM AM_API_RATINGS WHERE SUBSCRIBER_ID  = ? "
+                    + "AND API_PRODUCT_ID= ? ";
+
     public static final String GET_API_ALL_RATINGS_SQL =
             "SELECT RATING_ID, API_ID, RATING, SUBSCRIBER_ID FROM AM_API_RATINGS WHERE API_ID= ? ";
 
+    public static final String GET_API_PRODUCT_ALL_RATINGS_SQL =
+            "SELECT RATING_ID, API_PRODUCT_ID, RATING, SUBSCRIBER_ID FROM AM_API_RATINGS WHERE API_PRODUCT_ID= ? ";
+
     public static final String GET_SUBSCRIBER_NAME_FROM_ID_SQL =
             "SELECT USER_ID FROM AM_SUBSCRIBER WHERE SUBSCRIBER_ID = ? ";
 
     public static final String GET_RATING_INFO_BY_ID_SQL =
             "SELECT RATING_ID, API_ID, RATING, SUBSCRIBER_ID FROM AM_API_RATINGS WHERE RATING_ID = ? "
                     + "AND API_ID= ? ";
 
-    public static final String GET_AVERAGE_RATING_SQL =
+    public static final String REMOVE_FROM_API_RATING_SQL =
```

Comment:
Let's remove REMOVE_FROM_API_RATING_SQL and REMOVE_FROM_API_PRODUCT_RATING_SQL queries as they are not used.